### PR TITLE
docs: beef up readme, add quickstart

### LINKED ISSUE #208: project status / docs
what is the lifecycle status of this project?
how is this project being funded?
why isn't it on crates.io?
where are the docs for this project?
are their any benchmarks compared to kubo for for large (image, video) files?

### LINKED ISSUE #210: Entry point into iroh
Hello everyone,

I stumbled upon this project in my search for a rust ipfs implementation. However, I couldn't find a good entry point into iroh, as there is no docs.rs documentation (or any other documentation for that matter if I understand correctly) and no tutorials. Is there a good starting point besides just attempting to read source code directly? Maybe a simple example of fetching a file from ipfs or something like that?

### PR DESCRIPTION
**Pull request by b5**
we're not ready to launch a full docs site yet, so let's add a bit more to the readme, and move the quickstart guide into the repo while we work toward a release

closes #208, closes #210

### CODE REVIEW

File: quickstart.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+## 1. Run Iroh locally
+
+we’ve prepped a bash file one-liner for you:
+
+```bash
+$ curl -fsSL http://sh.iroh.computer/setup.sh | sh -s -- quickstart
+```
+
+Running stuff from the internet directly on your terminal is generally a bad idea. Feel free to read through the script first 😄 by dropping the pipe `|` character and everything after it to print `[setup.sh](http://setup.sh)` to your terminal. The setup script takes a bit of time to fetch precompiled binaries. Once up and running you should see terminal output that looks like this:
```
Comment (fabricedesre) [inline quickstart.md:17]: https://setup.sh is not the correct url?

File: quickstart.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+starting iroh-gateway.
+iroh-gateway started
+view logs at ~/.iroh/loq/iroh-gatewau.log
+iroh started
+iroh-gateway available at http: //localhost:9050
+you can run iroh-ctl from ~/.iroh/bin/iroh-ctl
+```
+
+
+🚧 *currently [setup.sh](http://setup.sh) writes everything to `$HOME/.iroh`, Iroh cloud will be switching to use [standard directories](https://dirs.dev/) for config, cache, and application data. See [issue #142](https://github.com/n0-computer/iroh/issues/142) for more detail:*
```
Comment (fabricedesre) [inline quickstart.md:40]: Same here

File: quickstart.md (hunk 1 above)
Comment (Arqu) [reply quickstart.md:17]: Also `https` not `http`

File: quickstart.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+```bash
+$ curl -fsSL http://sh.iroh.computer/setup.sh | sh -s -- quickstart
+```
+
+Running stuff from the internet directly on your terminal is generally a bad idea. Feel free to read through the script first 😄 by dropping the pipe `|` character and everything after it to print `[setup.sh](http://setup.sh)` to your terminal. The setup script takes a bit of time to fetch precompiled binaries. Once up and running you should see terminal output that looks like this:
+
+```
+$ curl -fsSL http://sh.iroh.computer/setup.sh | sh-s--quickstart
+Fetching http: //vorc.gekko.sh:9055/bin/iroh-gateway/darwin/aarch64/latest
```
Comment (Arqu) [inline quickstart.md:21]: It's now moved to `https://vorc.iroh.computer/bin/...`

File: quickstart.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+we’ve prepped a bash file one-liner for you:
+
+```bash
+$ curl -fsSL http://sh.iroh.computer/setup.sh | sh -s -- quickstart
+```
+
+Running stuff from the internet directly on your terminal is generally a bad idea. Feel free to read through the script first 😄 by dropping the pipe `|` character and everything after it to print `[setup.sh](http://setup.sh)` to your terminal. The setup script takes a bit of time to fetch precompiled binaries. Once up and running you should see terminal output that looks like this:
+
+```
+$ curl -fsSL http://sh.iroh.computer/setup.sh | sh-s--quickstart
```
Comment (Arqu) [inline quickstart.md:20]: `https://sh.iroh.computer/setup.sh`

File: quickstart.md [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+```
+
+This indicates all three services are running & healthy.
+
+## 4. Stopping iroh
+
+This setup script includes a few options, which you can see by replacing `quickstart` with `-h`:
+
+```bash
+$ curl -fsSL http://sh.iroh.computer/setup.sh | sh -s -- -h
```
Comment (Arqu) [inline quickstart.md:65]: same as above

**Comment by b5** [comment]
Thanks for the feedback all. Ready for re-review

File: quickstart.md [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+Fetching https://vorc.iroh.computer/bin/iroh-ctl/darwin/aarch64/latest
+starting iroh-store.
+iroh-store started
+view logs at ~/.iroh/log/iroh-store.log
+starting iroh-p2p.
+iroh-p2p started
+view logs at ~/.iroh/log/iroh-p2p.log
+starting iroh-gateway.
+iroh-gateway started
+view logs at ~/.iroh/loq/iroh-gatewau.log
```
Comment (Arqu) [inline quickstart.md:33]: `view logs at ~/.iroh/loq/iroh-gatewau.log` > `view logs at ~/.iroh/log/iroh-gateway.log`

**Review by Arqu — COMMENTED** [review]
Just a nitpick and a note that the quicksetup currently only works on linux/mac, otherwise you have to build for your arch and there are currently no guarantees.

**Comment by b5** [comment]
@Arqu yes there's a note in the quickstart that it only works for linux & mac, line 6 of `quickstart.md`

**Review by Arqu — APPROVED** [review]
🚀

**Pull request merged**